index.html

```diff
@@ -27,19 +27,4 @@
   app.mount('#app');
 </script>
 <style>
-
-.bg-purple {
-  background-color: rgb(41, 45, 62);
-}
-.callout-warning {
-    border-left-color: #f0ad4e!important;
-}
-.callout {
-    padding: 1.25rem;
-    margin-top: 1.25rem;
-    margin-bottom: 1.25rem;
-    border: 1px solid #e9ecef;
-    border-left-width: 0.25rem;
-    border-radius: 0.25rem;
-}
-</style>
```

Comment:
we need this classes, where were they deleted?

Reply:
added these css classes to index.html